tests/test_failure_message.py

```diff
@@ -2,6 +2,21 @@
 from test_results_parser import build_message, shorten_file_paths
 
 
+@dataclass
+class Thing:
```

Comment:
Potentially beyond the scope, but what is thing here? And could we rename this to what it's meant to mean? 🧠  (unless we're actually calling something "thing")

Reply:
Good point. I should've renamed it when I refactored it.